python/python/glide/async_commands/transaction.py

```diff
@@ -2036,6 +2037,105 @@ def function_stats(self: TTransaction) -> TTransaction:
         """
         return self.append_command(RequestType.FunctionStats, [])
 
+    def function_dump(self: TTransaction) -> TTransaction:
+        """
+        Returns the serialized payload of all loaded libraries.
+
+        See https://valkey.io/commands/function-dump/ for more details.
+
+        Command response:
+            bytes: The serialized payload of all loaded libraries.
+
+        Since: Redis version 7.0.0.
+        """
+        return self.append_command(RequestType.FunctionDump, [])
+
+    def function_restore(
+        self: TTransaction,
+        payload: TEncodable,
+        policy: Optional[FunctionRestorePolicy] = None,
+    ) -> TTransaction:
+        """
+        Restores libraries from the serialized payload returned by the `function_dump` command.
+
+        See https://valkey.io/commands/function-restore/ for more details.
+
+        Args:
+            payload (TEncodable): The serialized data from the `function_dump` command.
+            policy (Optional[FunctionRestorePolicy]): A policy for handling existing libraries.
+
+        Command response:
+            TOK: A simple "OK" response.
+
+        Since: Redis version 7.0.0.
+        """
+        args: List[TEncodable] = [payload]
+        if policy is not None:
+            args.append(policy.value)
+        return self.append_command(RequestType.FunctionRestore, args)
+
+    def dump(
+        self: TTransaction,
+        key: TEncodable,
+    ) -> TTransaction:
+        """
+        Serialize the value stored at `key` in a Valkey-specific format and return it to the user.
+
+        See https://valkey.io/commands/dump/ for more details.
+
+        Args:
+            key (TEncodable): The `key` to serialize.
+
+        Command response:
+            Optional[bytes]: The serialized value of the data stored at `key`. If `key` does not
+                exist, `None` will be returned.
+
+        Since: Redis version 7.0.0.
```

Comment:
the docs say this command has been available since 2.6, so I think we can remove this line
```suggestion
```

Reply:
Copy and paste error. Nice catch.